src/components/blog/BlogOverview.jsx

```diff
@@ -1,5 +1,37 @@
 import React from "react";
+import { Container, Row } from "react-bootstrap";
+import SingleBlogCard from "./SingleBlogCard";
+import "./style.overview.scss";
+import BlogHeader from "./BlogHeader";
 
 export default function BlogOverview() {
-  return <div></div>;
+  const [allPosts, setAllPosts] = React.useState([]);
+
+  const getAllPosts = async () => {
+    const data = await fetch(
+      "https://public-api.wordpress.com/wp/v2/sites/ecommerce458750409.wordpress.com/posts"
```

Comment:
are the same fetches being used in both your and Mira's PR? what's the difference?

delete console line 18

Reply:
yes the same fetches are being used in both branches